app/assets/javascripts/admin/addon/components/plugin-commit-hash.hbs

```diff
@@ -1,6 +1,8 @@
 {{#if this.commitHash}}
   <a
-    href={{this.plugin.commit_url}}
```

Comment:
Hmm why not just `@plugin.commit_url` instead of having to set an alias?

Reply:
You don't need `this.args.plugin` though, you can refer to it as `@plugin`.

Reply:
oh okay, thanks!